data/korath/korath missions.txt

```diff
@@ -257,3 +257,54 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Korath Boarding Response"
+	invisible
+	boarding
+	repeat
+	minor
+	deadline 1
+	to offer
+		not "Korath Boarding Response: active"
+		or
+			and
+				random < 5
+				"reputation: Korath" < -10
+				"reputation: Korath" >= -100
+			and
+				random < 10
+				"reputation: Korath" < -100
+				"reputation: Korath" >= -1000
+			and
+				random < 25
+				"reputation: Korath" < -1000
+				"reputation: Korath" >= -10000
+			and
+				random < 50
+				"reputation: Korath" < -10000
+
+			and
+				random < 10
+				"reputation: Korath (Civilian)" < 0
+				"reputation: Korath (Civilian)" >= -300
+			and
+				random < 25
+				"reputation: Korath (Civilian)" < -300
+				"reputation: Korath (Civilian)" >= -3000
+			and
+				random < 50
+				"reputation: Korath (Civilian)" < -3000
+	source
+		government Korath "Korath (Civilian)"
+		system Atik Durax Misam
```

Comment:
```suggestion
		system Atik Durax Misam Alcyone Almach
```

Reply:
I think it makes sense for the Korath boarding response to happen in pirate systems since one of the main goals is to deter pirates from pillaging Korath vessels. These planets are sufficiently out-of-the-way, and filled with hostile fleets, that a few strange hostile ships won't be a major new story.